src/crimson/osd/osd_operations/osdop_params.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "messages/MOSDOp.h"
+
+struct OSDOpParams {
```

Comment:
could you add a short comment to explain what this struct is for? also, i'd suggest use `osd_op_params_t` for data only class.

Reply:
also, to me it's more like `transaction_params` instead of `op_params`.

Reply:
The fields in this struct are parameters that may be needed in multiple level of processing, like PGBackend::mutate_object and ReplicatedBackend::_submit_transaction. I inclosed all those parameters in this struct to avoid passing each of them as a method parameter.

Reply:
by comment, i mean a comment in the code to explain what this struct is for.

Reply:
Oh, yes, sorry